### PR TITLE
Fluentd support

### PR DESCRIPTION
**Pull request by mdelaossa**
For now meant to be used as a separate image. Currenly published as `fluentd` tag.

I've removed stunnel as we can now use fluentd's SSL support directly.

I've also removed redis as we do not support tail logdrains with fluentd, at least not for now.

### CODE REVIEW

File: bin/run-gentleman-jerry.sh [hunk 1]
```diff
@@ -20,57 +20,19 @@ if [[ -n "$SSL_KEY" ]]; then
   mkdir -p "$(dirname "$SSL_KEY_FILE")"
   touch "$SSL_KEY_FILE"
   chmod 600 "$SSL_KEY_FILE"
-  echo "$SSL_KEY" > "$SSL_KEY_FILE"
+  echo "$SSL_KEY" >"$SSL_KEY_FILE"
```
Comment (almathew) [inline bin/run-gentleman-jerry.sh:23]: Unimportant, but why delete the space?

**Review by almathew — APPROVED** [review]
Same deal as JC - this looks good as a first pass, but we should update the tag for JC + GJ to use a consistent tag that isn't latest, v1, or v2

File: bin/run-gentleman-jerry.sh (hunk 1 above)
Comment (mdelaossa) [reply bin/run-gentleman-jerry.sh:23]: oh, this was unintentional, probably just a stray backspace

**Comment by mdelaossa** [comment]
Please check commit https://github.com/aptible/gentlemanjerry/pull/48/commits/91e28847cf6e1ec38619547888b592cf3977909e for updated tests

**Review by almathew — APPROVED** [review]

File: Dockerfile.erb [hunk 2]
```diff
@@ -1,84 +1,38 @@
...
 
-ENV JDK_VERSION openjdk7
-RUN apk update && apk add --no-cache bash curl "${JDK_VERSION}-jre-base" ruby java-cacerts
+USER root
 
 # Add extra certificates. Specifically, rapidssl is needed because it's used
 # by Papertrail.
 ENV LOCAL_TRUST_DIR /usr/local/share/ca-certificates
-ADD rapidssl.crt "$LOCAL_TRUST_DIR/rapidssl.crt"
+COPY rapidssl.crt "$LOCAL_TRUST_DIR/rapidssl.crt"
```
Comment (almathew) [inline Dockerfile.erb:8]: This is part of what I was talking about in [this Slack message](https://aptible-support.slack.com/archives/C022BUSK487/p1622814695034700?thread_ts=1622811611.031600&cid=C022BUSK487), but this probably isn't necessary anymore?
I'm assuming it's included in the papertrail plugin?

**Review by almathew — APPROVED** [review]

File: Dockerfile.erb (hunk 2 above)
Comment (mdelaossa) [reply Dockerfile.erb:8]: ah, I see what you mean! It is most likely included in the papertrail plugin, but I won't feel comfortable removing it without testing first

**Pull request merged**